airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -704,6 +703,16 @@ class PullRequestStats(PullRequestSubstream):
     API docs: https://docs.github.com/en/rest/reference/pulls#get-a-pull-request
     """
 
+    use_cache = True
```

Comment:
why we enabled `use_cache` here ? it's knob for VCR library as I remember

Reply:
good call, removed it, because we already have cache set via requests-cache library